docs/Modules/bpmn-modules/modeling.md

```diff
@@ -9,4 +9,160 @@ parent: "BPMN Modules"
 grand_parent: "Modules"
 ---
 
-# modeling
\ No newline at end of file
+# modeling
+
+This module activates the BPMN 2.0 modeling features [over the common modeling module](/modules/common-modules/modeling). It means that this module contains every API exported by the common modeling module.
+
+Additional APIs over the common modeling module are:
+
+* [updateLabel](#updatelabel)
+* [connect](#connect)
+* [updateProperties](#updateproperties)
+* [resizeLane](#resizelane)
+* [addLane](#addlane)
+* [splitLane](#splitlane)
+* [setColor](#setcolor)
+
+## updateLabel
```

Comment:
I miss a bit the general input parameter and return value statements to use this API. Or do we describe them in the separate command sections?

Reply:
Parameters of each method should be documented.

Reply:
Done 👍